README.md

```diff
@@ -47,6 +49,20 @@ require('ayu').setup({
 })
 ```
 
+2. Change the background color of non-active windows to make the active one more obvious, specifying overrides for both light and dark backgrounds:
+
+```lua
+require 'ayu'.setup({
+  overrides = function()
+    return vim.o.background == 'dark' and
+          { NormalNC = {bg = '#0f151e', fg = '#808080'} }
+          or
+          { NormalNC = {bg = '#f0f0f0', fg = '#808080'} }
```

Comment:
Could you use regular if then else here? In my opinion it will look nicer for this case.